bom/pom.xml

```diff
@@ -33,8 +33,8 @@
   <name>Jakarta EE TCK Artifacts BOM</name>
 
   <properties>
-    <version.jakarta.tck.arquillian>11.0.0-RC1</version.jakarta.tck.arquillian>
-    <version.jakarta.tck.common>11.0.0-RC5</version.jakarta.tck.common>
+    <version.jakarta.tck.arquillian>11.0.0-RC2</version.jakarta.tck.arquillian>
```

Comment:
We should probably rename this property to `jakarta.tck.arquillian.version` to be consistent with the others.